frontend/public/components/secrets/create-secret/PullSecretForm.tsx

```diff
@@ -0,0 +1,109 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Dropdown } from '../../utils';
+import {
+  SecretType,
+  getImageSecretKey,
+  AUTHS_KEY,
+  CreateConfigSubform,
+  UploadConfigSubform,
+} from '.';
+import { useCallback, useState } from 'react';
+
+export const PullSecretForm: React.FC<PullSecretFormProps> = ({
```

Comment:
Import new SecretSubFormProps and use them here
```suggestion
export const PullSecretForm: React.FC<SecretSubFormProps> = ({
```